README.md

```diff
@@ -46,7 +47,7 @@ All modifications to this repository are released under the same terms as [liboq
 
 ## Team
 
-The Open Quantum Safe project is led by [Douglas Stebila](https://www.douglas.stebila.ca/research/) and [Michele Mosca](http://faculty.iqc.uwaterloo.ca/mmosca/)at the University of Waterloo.
+The Open Quantum Safe project has been taken over by LinuxFoundation within [PQCA](https://pqca.org). 
```

Comment:
```suggestion
The Open Quantum Safe project is part of the Linux Foundation within [PQCA](https://pqca.org). 
```

Reply:
This change would completely eliminate the history of OQS as a free/"meritocratic", non-LF project. If you don't like the term "taken over" what about "After 7 years of stewardship by UWaterloo, control of the project changed to the newly established PQCA, a Linux Foundation alliance."?

Reply:
This is documented to my satisfaction at https://openquantumsafe.org/news/, so I don't have the need to repeat it in each README.  But if we do want to include it in each README, then I'd just copy the text from the website.

Reply:
> then I'd just copy the text from the website

Sounds a bit 20th century :-) What about instead of the text simply linking (actually from/for all sub project's README's) to there? This may actually remove lots of redundant text (e.g., regarding funding) in many sub projects.

Reply:
Sure, we can do that.

Reply:
Added link to OQS news.